Source/Scene/Cesium3DTileStyleEngine.js

```diff
@@ -113,6 +115,12 @@ define([
             return;
         }
 
+        //>>includeStart('debug', pragmas.debug);
+        if (!style.ready) {
```

Comment:
Move this check to inside `evaluateColor` and `evaluate`.  That is the object that isn't ready, not this.